packages/paypal-commerce-integration/src/paypal-commerce-ratepay/paypal-commerce-ratepay-payment-strategy.ts

```diff
@@ -189,14 +189,16 @@ export default class PaypalCommerceRatepayPaymentStrategy implements PaymentStra
     }
 
     private normalizeDate(date: BirthDate) {
-        const formattedDate =
-            date.getDate() < 10 ? `0${date.getDate()}` : date.getDate().toString();
-        const formattedMonth =
-            date.getMonth() < 10 ? `0${date.getMonth() + 1}` : date.getMonth().toString();
+        const formattedDate = this.formatDate(date.getDate());
+        const formattedMonth = this.formatDate(date.getMonth() + 1);
 
         return `${date.getFullYear()}-${formattedMonth}-${formattedDate}`;
     }
 
+    private formatDate = (date: number): string => {
```

Comment:
It might be better to have the same styling as other methods have. Otherwise, you can place the method inside the function where you are using it

Reply:
refactored